From 48cd822978fdaa3452eec0181629116736cba24b Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Thu, 26 Jul 2018 16:09:08 +0800
Subject: [PATCH] ARM: dts: rockchip: rk3308-voice-module: enable 200ms delay
 play property

Due to the PA CS8509 on rk3308 voice module startup time
is ~200ms, we need to hold it some times, avoid lossing
audio data during start playback.

Change-Id: I44ac3caa0ceddbe7720d8eb63179ae56baa168a9
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi b/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
index ef8c46537693..92a27328cd32 100644
--- a/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
+++ b/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
@@ -182,6 +182,7 @@
 	status = "disabled";
 
 	rockchip,no-hp-det;
+	rockchip,delay-start-play-ms = <200>;
 	rockchip,loopback-grp = <1>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
 };
-- 
2.35.3

